Updates basic tutorials for Cadence 1.0 and fixes old migration commands (#112)
onflow/cadence-lang.org: versioned_docs/version-1.0/cadence-migration-guide/index.md

#### Expectations

- Any contracts that are not upgraded will fail after Testnet and Mainnet upgrade.

>   They **WILL NOT** be able to be upgraded again, so it is imperative

Reviewer: ```suggestion
  They **cannot** be upgraded again, so it is imperative
```
